GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml

```diff
@@ -0,0 +1,195 @@
+                    <div class="govuk-body">
+                        No organisations
+                    </div>
+                </div>
+            </details>
+        </div>
+    </div>
+
+    <div>
+        <h2 class="govuk-heading-l">Organisation Sector Types</h2>
+
+        <div id="PublicSectorOrganisationsWithoutAPublicSectorType">
+            <details class="govuk-details" data-module="govuk-details">
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                        <span class="govuk-!-font-weight-bold govuk-!-font-size-36">
+                            Loading...
+                        </span>
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                    <div class="govuk-body">
+                        No organisations
+                    </div>
+                </div>
+            </details>
+        </div>
+    </div>
+
+    <script type="text/javascript">
+        document.addEventListener("DOMContentLoaded", function() { 
+            $.get("@Url.Action("OrganisationsWithMultipleActiveAddressesAjax", "AdminDatabaseIntegrityChecks")",
+                "",
+                function(data) { $("#OrganisationsWithMultipleActiveAddresses").html(data)
+                },
+                "html"
+              );
+            
+            $.get("@Url.Action("OrganisationsWhereLatestAddressIsNotActiveAjax", "AdminDatabaseIntegrityChecks")",
+                "",
+                function(data) { $("#OrganisationsWhereLatestAddressIsNotActive").html(data)
+                },
+                "html"
+              );
+
+            $.get("@Url.Action("OrganisationsWithMultipleActiveScopesForASingleYearAjax", "AdminDatabaseIntegrityChecks")",
+                "",
+                function(data) { $("#OrganisationsWithMultipleActiveScopesForASingleYear").html(data)
+                },
+                "html"
+              );
+            
+            $.get("@Url.Action("OrganisationsWithNoActiveScopeForEveryYearAjax", "AdminDatabaseIntegrityChecks")",
+                "",
+                function(data) { $("#OrganisationsWithNoActiveScopeForEveryYear").html(data)
+                },
+                "html"
+              );
+            
+            $.get("@Url.Action("OrganisationsWithMultipleActiveReturnsForASingleYearAjax", "AdminDatabaseIntegrityChecks")",
+                "",
+                function(data) { $("#OrganisationsWithMultipleActiveReturnsForASingleYear").html(data)
+                },
+                "html"
+              );
+            
+            $.get("@Url.Action("PublicSectorOrganisationsWithoutAPublicSectorTypeAjax", "AdminDatabaseIntegrityChecks")",
+                "",
+                function(data) { $("#PublicSectorOrganisationsWithoutAPublicSectorType").html(data)
+                },
+                "html"
+              );
```

Comment:
So it looks like you can't pass js variables into Razor as Razor is server-side and gets converted to html/js by the time you want to pass in your variable. So the way I have written the function for now is 

```
    function loadHtml(url, elementName) {
        $.get("/admin/" + url ,
        "",
        function(data) { $("#" + elementName).html(data)},
        "html"
        );
    }

    document.addEventListener("DOMContentLoaded", function() { 
        loadHtml("active-organisations-with-the-same-name-ajax", "ActiveOrganisationsWithTheSameName");    
    });

```

You can't use `@(Url.Action)` which would have allowed us to use the Action name which is Pascal case rather than kebab-case so that we wouldn't need to enter the same name twice into the function. All of this is a bit sad and I'd be very open to suggestions and improvements.